_security-plugin/access-control/document-level-security.md

```diff
@@ -55,6 +55,36 @@ PUT _plugins/_security/api/roles/public_data
 These queries can be as complex as you want, but we recommend keeping them simple to minimize the performance impact that the document-level security feature has on the cluster.
 {: .warning }
 
+### A note on field values and special characters
+
+Due to word boundaries associated with Unicode special characters, the Unicode standard analyzer cannot interpret the portion of an index field value that follows a special character. Therefore, special characters should not be used for values in either query DSL fields or REST API query fields unless a custom analyzer is used. The examples below illustrate values that will be misinterpreted as the same when a special character – such as the hyphen/minus sign – exists in the value:
```

Comment:
Standard analyzer does interpret the portion after special character. The values are basically tokenized. For e.g. `User-1` below will be indexed as two separate values i.e. `User` and `1`. The same behavior analyzer is used during the search queries (until unless a separate analyzer is provided in search query - not recommended).

Reply:
This accounts for the difference between indexing strings vs indexing integers. I think what you're saying @dreamer-89 the analyzer will interpret `1` but not `User`. Is that correct?

That being said, should we include a list of the most common characters that should be avoided?